cocos/render-scene/core/render-window.ts

```diff
@@ -103,6 +150,14 @@ export class RenderWindow {
     protected _hasOffScreenAttachments = false;
     protected _framebuffer: Framebuffer | null = null;
     protected _device: Device | null = null;
+    // _renderWindowId is used to identify the render window in the render pipeline
+    protected _renderWindowId = _windowCount++;
+    // _isResized is used to indicate whether the render window is resized
+    protected _isResized = true;
```

Comment:
Why default value is true?

Reply:
When render window is initialized, we consider it is resized. So the resize handler can be triggered and user can register resources in the handler.

Reply:
Got it.